oaknational/Oak-Web-Application
# feat: update link to legacy curriculum maps

## Description

Music year: {owa_music_year}

- Change legacy curriculum link in Subject Phase Picker to point previous downloads page

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2250--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Click on `Subject` control
3. Click on `Previously Released Plans`
4. You should navigate to the "Previously released curricula" page with the path `/teachers/curriculum/previous-downloads`

## Checklist
